code/modules/assembly/voice.dm

```diff
@@ -10,46 +10,67 @@
 
 	bomb_name = "voice-activated bomb"
 
-	describe()
```

Comment:
Could you please comment where you made changes to this file? While the absolute pathing is completely appreciated (thank you), it does make it a bit harder for us to pick out what was changed.

Reply:
Line 63 and below, everything above was just pathing so it would be cleaner to add /noise